.github/workflows/main.yml

```diff
@@ -582,3 +582,18 @@ jobs:
         files: "dist/*"
         name: ${{ steps.tagname.outputs.val }}
         token: ${{ secrets.GITHUB_TOKEN }}
+
+  cargo-audit:
+    env:
+      CARGO_AUDIT_VERSION: 0.11.2
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - uses: actions/cache@v1
+      with:
+        path: ${{ runner.tool_cache }}/cargo-audit
+        key: cargo-audit-bin-${{ env.CARGO_AUDIT_VERSION }}
+    - run: echo "::add-path::${{ runner.tool_cache }}/cargo-audit/bin"
+    - run: |
+        cargo install --root ${{ runner.tool_cache }}/cargo-audit cargo-audit
```

Comment:
I think this will want to use `--vers` to install the version in `CARGO_AUDIT_VERSION`?

Reply:
Oops, forgot about that. Fixed in the recent push.